Description: Change mathjax_path
 _static/mathjax are symlinks to /usr/share/javascript/mathjax created in debian/rules.
Author: Tobias Hansen <thansen@debian.org>
Forwarded: not-needed

--- a/sage/src/sage/docs/conf.py
+++ b/sage/src/sage/docs/conf.py
@@ -246,7 +246,7 @@
 
 # We use MathJax to build the documentation.
 extensions.append('sphinx.ext.mathjax')
-mathjax_path = 'MathJax.js?config=TeX-AMS_HTML-full,../mathjax_sage.js'
+mathjax_path = 'mathjax/MathJax.js?config=TeX-AMS_HTML-full,../../mathjax_sage.js'
 
 from sage.misc.latex_macros import sage_mathjax_macros
 html_theme_options['mathjax_macros'] = sage_mathjax_macros()
